corehq/apps/appstore/management/commands/export_exchange.py

```diff
@@ -0,0 +1,54 @@
+import csv
+
+from datetime import datetime
+from django.core.management.base import BaseCommand
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import CouchUser
+from dimagi.utils.parsing import ISO_DATETIME_FORMAT
+
+
+class Command(BaseCommand):
+    help = """
+        Pulls data on domain snapshots (projects in the late CommCare Exchange).
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('outfile')
+
+    def handle(self, outfile, **options):
+        snapshots = Domain.get_db().view(
+            'domain/snapshots',
+            startkey=[],
+            endkey=[{}],
+            reduce=False,
+            include_docs=True,
+        ).all()
+        snapshots = [s['doc'] for s in snapshots]
+        snapshots = [s for s in snapshots if s['published'] and s['is_snapshot'] and s.get('snapshot_head')]
```

Comment:
why keep these separate?

Reply:
To avoid line length limits and weird indentation :-p